src/references/rainbow-token-list/index.ts

```diff
@@ -124,15 +133,15 @@ async function getTokenListUpdate(
     const freshDate = new Date((data as TokenListData)?.timestamp);
     if (freshDate > currentDate) {
       const work = [
-        writeRNFSJsonData<TokenListData>(
+        writeMMKVJsonData<TokenListData>(
```

Comment:
if this is not async (and is not, this only pretending to be async), the whole workers logic is redundant

Reply:
Seconding what Michał wrote.